style.css

```diff
@@ -421,12 +421,20 @@ body {
   border: 1px solid var(--color-sccent);
 }
 
+.popup-btn svg path {
+  fill: var(--color-white);
+}
+
```

Comment:
- Great Job for implementing the functionality 👍 however you need to adjust the font, size of buttons, Mobile Poup margin, and the gap between image and description in your design to stick to the Figma design and have better UI 👌
### Check the Bellow Images for more info 👇👇

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/231593321-d2782df6-49c3-418c-8528-4e6c915a9f82.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/231593319-4415585b-6559-4b51-936b-b8ef7c9deea0.png" width="200">|


😎️

🤝️
### Figma Design 👇
![2023-04-13_00-40](https://user-images.githubusercontent.com/52010789/231593307-f151ceea-d501-444e-a11c-0405f14fdf23.png)
### Your Design 👇
![2023-04-13_00-42](https://user-images.githubusercontent.com/52010789/231593313-9f2defe8-25f1-4918-acf6-9681acaf5d8d.png)

Reply:
I did not focus on the styles because I thought this project is about functionality more than anything else.

Reply:
@binhussen, I did not focus on the styles because I thought this project is about functionality more than anything else.

Reply:
Great job on making the adjustments here 💯 but make sure to keep some margin between the popup and its sides, there's currently no space at all on its left and right, take a look 👇🏽 

![Screenshot 2023-04-12 at 22 12 56](https://user-images.githubusercontent.com/45739963/231629163-6c8930fd-768e-4f59-aaab-86bf8772804f.png)

And on the mobile version because of the same margin issue, it becomes impossible to reach the ❌  close button to close the popup 😢 

![Screenshot 2023-04-12 at 22 13 40](https://user-images.githubusercontent.com/45739963/231629252-d415cfa0-42cb-42fe-be59-9ad972a40d20.png)

Also, there seems to be some overflow issue with your title here, take a look at what I get when I scroll to the right 👇🏽 

![Screenshot 2023-04-12 at 22 19 04](https://user-images.githubusercontent.com/45739963/231630363-61d57768-2432-4367-b23b-17bfe0e7a120.png)

Reply:
Can you explain more ? do you mean add padding to the container?

Reply:
@OTMANE-ECHCHAFYKY padding means the space is inside the container, I think margin instead, the space is outside the container.